CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+<!--- Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com> -->
+# Lagom contributor guidelines
+
+## Prerequisites
+
+Before making a contribution, it is important to make sure that the change you wish to make and the approach you wish to take will likely be accepted, otherwise you may end up doing a lot of work for nothing. The Lagom site is the sum of several sources, make sure the change you are working on happens in the proper repository. In case of doubt, feel free to open an [issue](https://github.com/lagom/lagom.github.io/issues) to discuss the proper approach.
+
+## Development tips
+
+- See [README.md](README.md) for instructions on running the Lagom site locally.
+- The Lagom site is the materialization of several sources: [lagom.github.io's _template_ branch](https://github.com/lagom/lagom.github.io/tree/template), [Lagom's `docs/` folder](https://github.com/lagom/lagom/tree/master/docs) and Lagom's API docs for both Java and Scala as generated from the sources in the [Lagom repository](https://github.com/lagom/lagom). All information coming from the [Lagom repository](https://github.com/lagom/lagom) is created from several branches so that many versions of the docs can be maintained.
+
+## Pull request procedure
```

Comment:
My suggestion is:

Add a paragraph (maybe a subhead also)

Contributing to the documentation

To fix an issue in the documentation or add to an existing page, go to the live page on http://www.lagomframework.com/documentation/ and click the link to the source code at the bottom of the page. 

To contribute larger documentation changes, for example, adding pages, you might prefer to edit the source files in a forked repository of https://github.com/lagom/lagom/ and create a Pull Request. You'll find the documentation source files  ...lagom/docs/manual.